src/components/SingleDatePicker.jsx

```diff
@@ -253,25 +253,25 @@ class SingleDatePicker extends React.PureComponent {
   }
 
```

Comment:
The changes in this file seem odd, but they are just moving the calls for ` (add|remove)EventListeners ` and having them use dayPickerContainer instead.